packages/lib/actionClass/service.ts

```diff
@@ -94,7 +102,7 @@ export const getActionClass = async (actionClassId: string): Promise<TActionClas
           select,
         });
 
-        return actionClass;
+        return formatDateFields(actionClass, ZActionClass) as TActionClass;
```

Comment:
Can we remove the `as TActionClass` by setting to return type of `formatDateFields` correctly? (see comment in datetime.ts)